models/VersionInformation.ts

```diff
@@ -0,0 +1,42 @@
+import { VersionInformationInvalid, VersionInformationMissing } from '../errors/Errors';
+
+function isDefined(value: any): boolean {
+    return value !== null && value !== undefined;
+}
+
+/**
+ * Defines version information for a controller or a route. Throws if there are errors with the information.
+ * 
+ * @class VersionInformation
+ */
+export class VersionInformation {
+    public static create(name: string, versionInformation: { from?: number, until?: number }): VersionInformation {
+        if (!(isDefined(versionInformation.from) || isDefined(versionInformation.until))) {
+            throw new VersionInformationMissing(name);
+        }
+
+        if (isDefined(versionInformation.from) && (versionInformation.from.constructor !== Number || versionInformation.from < 1 || versionInformation.from % 1 !== 0)) {
+            throw new VersionInformationInvalid(name, `The from value (${versionInformation.from}) is either not a number, a floating point number or less than 1`);
+        }
+
+        if (isDefined(versionInformation.until) && (versionInformation.until.constructor !== Number || versionInformation.until < 1 || versionInformation.until % 1 !== 0)) {
```

Comment:
You could move `(versionInformation.until.constructor !== Number || versionInformation.until < 1 || versionInformation.until % 1 !== 0)` into a helper function since it is the same code as above...